packages/components/popover/package.json

```diff
@@ -22,6 +22,6 @@
     "directory": "packages/components/popover"
   },
   "dependencies": {
-    "@radix-ui/react-popover": "1.0.6-rc.13"
+    "@radix-ui/react-popover": "1.0.6"
```

Comment:
I'm always lost with version naming... Just to be sure: `1.0.6` > `1.0.6-rc.13`? Because the RC was specifically related to a bugfix https://github.com/radix-ui/primitives/issues/2128#issuecomment-1557314630